app/controllers/base_drafts_controller.rb

```diff
@@ -19,6 +19,11 @@ def show
     end
 
     add_breadcrumb breadcrumb_name(get_resource.draft, resource_name), send("#{resource_name}_path", get_resource)
+
+    respond_to do |format|
+      format.html { return }
```

Comment:
Is this line necessary?   if format matches html, won't it just return anyhow?

Reply:
The format.html { } is necessary or rails generates an unknown format error.  The return can be omitted.  I'd argue that { return } is more transparent than { }.

Reply:
Agree, thanks.